python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -130,6 +132,10 @@ def to_dagster_run_status(self) -> GrapheneRunStatus:
             return GrapheneRunStatus.STARTED  # pyright: ignore[reportReturnType]
         if self.args[0] == GrapheneBulkActionStatus.COMPLETED.value:  # pyright: ignore[reportAttributeAccessIssue]
             return GrapheneRunStatus.SUCCESS  # pyright: ignore[reportReturnType]
+        if self.args[0] == GrapheneBulkActionStatus.COMPLETED_SUCCESS.value:  # pyright: ignore[reportAttributeAccessIssue]
```

Comment:
I'd like to come up with a better solution for this conversion (if we maintain the `runStatus` graphene attr i mentioned in my other comment). It looks like we'll need to convert run statuses to bulk action statuses too for filtering, so i think a more complete solution is in order. Another thing that i can stack on, but isn't strictly required for this PR

Reply:
The two lines added here look fine to me. But wondering more generally what the filtering experience looks like -- if I filter to queued runs, do I see runs in the backfill? Or do I see the whole backfill display as a single run if any run is queued? Or something else?

Reply:
the scheme were going with for filtering is that the row-level object is what gets filtered. So if you are in the combined single runs + backfills view, and filter for queued status, you will only see single-runs that have queued status. and no backfills (even if the backfill has queued runs)